src/resources/scss/components/_zaa-image-array-upload.scss

```diff
@@ -12,4 +12,28 @@ zaa-image-array-upload {
     & + .flag {
         margin-right: 0;
     }
-}
\ No newline at end of file
+}
+
+.no-border{
```

Comment:
This changes has nothing do to with your PR, you should make PR only about a single topic. Please reset this file to master.